### ISSUE TITLE
Leave/mute DM conversation

### ISSUE THREAD

**Issue by staab** [coracle-social/coracle#122]
#70

**Comment by VictorieeMan** [comment]
Hm... There is already the option to mute accounts you see in the feeds. Where/how is this stored for each user? Is it stored on nostr relays, or within a client cookie?

**Comment by staab** [comment]
Those mutes are stored as kind 10000 events, muting DMs I see as a different use case.

**Comment by VictorieeMan** [comment]
Yes, I agree them to be a different use case. Muting an account in the DM context is not the same as in the feed context.
Does any NIP store mutes for DM? If not, the client needs to store some data for each user.

Might be a worthy NIP contribution, for DM_muting list, kind 9999 would be free and put the event type near the 10000 event; for feed mute list. Maybe @fiatjaf got an opinion here.
